src/forms/Textarea.jsx

```diff
@@ -112,7 +113,11 @@ export class Textarea extends React.PureComponent<Props, State> {
 
 		return (
 			<div className="inputContainer">
-				{label && <FieldLabel htmlFor={id}>{label}</FieldLabel>}
+				{label && (
+					<FieldLabel htmlFor={id} className={a11yPassThrough(labelClassName)}>
```

Comment:
Remove the `eslint-disable-line` rule on this line:
https://github.com/meetup/meetup-web-components/pull/704/files#diff-ed042334d974c5af7f7e9895abc95ddcR94